cloudformation/base.yaml

```diff
@@ -27,6 +27,11 @@ Parameters:
     Type: String
     Description: The name for your Cognito Identity Pool
 
+  AWSRegion:
```

Comment:
Not related to this commit - The S3 buckets above, **ArtifactsS3Bucket** and **DevPortalSiteS3Bucket** have an incorrect description. They say that bucket names are "region-unique" but they're actually [globally unique](https://docs.aws.amazon.com/AmazonS3/latest/dev/BucketRestrictions.html)